### DISCUSSION TITLE
在训练时，多个列表的数据在每个batch是如何获取数据的

### DISCUSSION THREAD

**Discussion by Dhao007** [PaddlePaddle/PaddleOCR#13233] [Q&A]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/130341825/a18a2f71-ef7e-4b4b-a10d-f018488ca3c3)
如图，我每个batch16张，是随机从列表获取16张还是均匀从每个列表获取

**Comment by GreatV** [comment]
建议看一下源代码
